### PR TITLE
[#152919324] Training info comparison between users

### PR DESCRIPTION
**Pull request by VincentHokie**
#### What does this PR do?
It allows users compare their training data with another user on the application, also, new graphs have been used that show all 3 dimensions of the data being represented

#### Any background context you want to provide?
Previously users could only see data belonging to them and had no way of gauging themselves against other users, this is now possible with this PR

#### What are the relevant pivotal tracker stories?
[#152919324](https://www.pivotaltracker.com/story/show/152919324)

#### Screenshots (if appropriate)
Previously, this is what the graphs looked like 
new

other

Now they look like this
screen shot 2017-12-06 at 09 30 31

data under a date can be seen by clicking on a date
screen shot 2017-12-04 at 10 31 21

and when comparing data, a user is selected from a dropdown list and the graph reloads to look something like this

screen shot 2017-12-06 at 09 30 42

an brief general interpretation of the graph has also been added to allow users easily understand the bubble graphs

screen shot 2017-12-08 at 11 35 39

### CODE REVIEW

**Review by CeciliaCaroline — COMMENTED** [review]
Good use of graphs to compare data of different users

**Review by Sharonsyra — COMMENTED** [review]
Nice looks better

**Review by jonathankamau — COMMENTED** [review]
Were you able to implement the changes from Monday's demo because I don't see that reflected on the screenshots.

**Review by jonathankamau — COMMENTED** [review]
Why 96 files changed?

File: .gitignore [hunk 1]
```diff
@@ -77,3 +77,9 @@ venv-django/
 
 #vscode settings
 .vscode/
+
+#migration files
+**/migrations/
+
+#MAC file system files
+**/.DS_Store
```
Comment (jonathankamau) [inline .gitignore]: New line required here.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Your build is failing on Travis CI. Please look into that.

File: .gitignore (hunk 1 above)
Comment (VincentHokie) [reply .gitignore]: Done.

**Comment by VincentHokie** [comment]
@jonathankamau the many files changed are mostly because of the ignored (and deleted in the git cache) migration files.

File: .travis.yml [hunk 2]
```diff
@@ -60,7 +64,9 @@ notifications:
...
+    secure: gG+IhVo2HtF2UgrEbUA+UcP1X9zr9XnCZeteYlAuSuGJ2zHkMU0qePeaCg6pMyk0QmrziV7JOGTX6eWIIWUEvykR98cCi3n7FCmb5fZSWS6hp7ak2VkabQ0g+XnWQsC9FeRk5NFcqrio+CGCcrCvf3soobZ6X1C8Pj0MCOlRVogTbYzMZ1krThZ3UPqbC4YlucbKdmRo7fi9X3lYrifuEvmT59xSNYW09Bg61b/sRm1q+OaBUMnGjQc8VixjHLElFJykHQ+pM/KZ11zdlm/59iAhrI3wRls7D7uNBQJwS9G43IEE9vLq0CT/n6cyxjcQzIM9J2JfizSmLsGutwy7FFjBRuTdSBali1ys0oNV/enQ0YuGW93N/h37GFHbXGkuU101IQ8JgubcMpd/xZ8QJOpt/rBJri0NF8dccY646H8HG7C8ZSSPKoimI3BuoNEw9N3EO9ElqMia2kneecCNARnLUnxZEABInzGt/C2o7ovkInnGHuoevYBCQcFkqrEV6M3nlXgiPIZEkAxtZaql5abiA0Di1+yQrAO6wniSQ/FuAfQxZ5r+CVymIpSqPe+aZ+VM4GYWyZrIWBr+duCEqaLMZsfB8SnHFAHtRZTDH72FtHK5NCJEzAjXmDSuip9DW/PB7cAd6W13fHcPy6XxUt2Ae3lwZLSUqVXpOQn21q4=
   app: wger-a-team
-  on: develop
-
+  run: "python manage.py makemigrations && python manage.py migrate"
+  on:
+    repo: andela/wg-a-team
+    branch: develop
```
Comment (jonathankamau) [inline .travis.yml]: New line needed here.

File: wger/core/templates/tags/render_weight_log.html [hunk 3]
```diff
@@ -1,32 +1,58 @@
...
             {% endfor %}
-        </table>
-    </td>
-    {% endfor %}
-    </tr>
-</table>
-</div>
+        </tr>
+    </table>
+</div>
```
Comment (jonathankamau) [inline wger/core/templates/tags/render_weight_log.html]: New line required here.

File: wger/manager/templates/workout/log.html [hunk 4]
```diff
@@ -9,112 +9,304 @@
...
+            $.get("/api/v2/workoutlog/?compare=comparison&limit=200&exercise="+exercise, function(data, status){
+                createGraph(data.results, uuid, myId, user_id);
+            });
+            
+        });
+
+    });
+</script>
+
+{% endblock %}
```
Comment (jonathankamau) [inline wger/manager/templates/workout/log.html]: New line needed here as well

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Add the Pivotal Tracker ID to the title of your PR in this format `#PTID Training info comparison between users` You don't need `Feature` as part of the title.

File: .travis.yml (hunk 2 above)
Comment (VincentHokie) [reply .travis.yml]: Done.

File: wger/core/templates/tags/render_weight_log.html (hunk 3 above)
Comment (VincentHokie) [reply wger/core/templates/tags/render_weight_log.html]: Done.

File: wger/manager/templates/workout/log.html (hunk 4 above)
Comment (VincentHokie) [reply wger/manager/templates/workout/log.html]: aaand done.

**Pull request merged**